networkx/algorithms/bipartite/matrix.py

```diff
@@ -133,7 +133,7 @@ def from_biadjacency_matrix(A, create_using=None, edge_attribute='weight'):
     0 or 1 representing membership in part 0 or part 1 of the bipartite graph.
 
     If `create_using` is an instance of :class:`networkx.MultiGraph` or
-    :class:`networkx.MultiDiGraph` and the entries of `A` are of type ``int``,
```

Comment:
Yes. If desired, the link can be disabled by prepending an exclamation point, so you'd just get the inline "code" formatting.